presentation/src/main/java/com/anytypeio/anytype/presentation/settings/SpacesStorageFactory.kt

```diff
@@ -0,0 +1,35 @@
+package com.anytypeio.anytype.presentation.settings
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.anytypeio.anytype.analytics.base.Analytics
+import com.anytypeio.anytype.domain.auth.interactor.GetAccount
+import com.anytypeio.anytype.domain.base.AppCoroutineDispatchers
+import com.anytypeio.anytype.domain.library.StorelessSubscriptionContainer
+import com.anytypeio.anytype.domain.workspace.InterceptFileLimitEvents
+import com.anytypeio.anytype.domain.workspace.SpaceManager
+import com.anytypeio.anytype.domain.workspace.SpacesUsageInfo
+import javax.inject.Inject
+
+class SpacesStorageFactory @Inject constructor(
```

Comment:
`SpacesStorageViewModelFactory` OR let's put it into VM

Reply:
done